examples/server/tests/features/issues.feature

```diff
@@ -0,0 +1,36 @@
+# List of ongoing issues
+@bug
+Feature: Issues
+    # Issue #5655
+  Scenario: Multi users embeddings
+    Given a server listening on localhost:8080
+    And   a model file stories260K.gguf
+    And   a model alias tinyllama-2
+    And   42 as server seed
+    And   64 KV cache size
```

Comment:
Apologies for pedantics, but KV cache size is measured in what? Some sort of domain-specific unit or is this just MB?